data/changelog/odoc/2019-03-08-odoc-1.4.0.md

```diff
@@ -20,7 +20,7 @@ changelog: |
   - Paragraph headings (`{4 ...}`) and subparagraph headings (`{5 ...}`) (#217,
     Rizo Isrof).
   - `odoc support-files-targets` command (#232).
-  - Recommend [`bsdoc`](https://ostera.github.io/bsdoc/docs/BsDoc/) for using
+  - Recommend [`bsdoc`](https://ostera.github.io/bsdoc/docs/BsDoc/ - [404 Not Found]) for using
```

Comment:
```suggestion
  - Recommend [`bsdoc`](https://rescript-lang.org/blog/bucklescript-is-rebranding) for using
```
BsDoc no longer available on Ostera's current GitHub page. This is the best I could find about it.

Reply:
I found this:

```suggestion
  - Recommend [`bsdoc`](https://github.com/reasonml-community/bsdoc) for using
```